frontend/src/scenes/debug/HogQLDebug.tsx

```diff
@@ -80,7 +80,25 @@ export function HogQLDebug({ query, setQuery, queryKey }: HogQLDebugProps): JSX.
                             }
                             value={query.modifiers?.inCohortVia ?? response?.modifiers?.inCohortVia}
                         />
-                    </LemonLabel>{' '}
+                    </LemonLabel>
+                    <LemonLabel>
```

Comment:
Can we add a minimum width on this so that all options are readable in the dropdown when its defaulted onto `auto`?

Reply:
In some ways sure, in some ways I'm reluctant because I don't know of any way to just make the select dropdown normal, without adding a lot of whitespace to the default small "auto" option. 
<img width="872" alt="image" src="https://github.com/PostHog/posthog/assets/53387/c2c2c1cb-fc60-4cbd-887e-fe66b05ed868">

This entire row is getting crowded, and we're surely going to add many more things, so this requires a bit of a rethink 🤔

Reply:
I merged it since all tests were green, and such alignment needs to be used. Happy to revisit this later.